src/app/_components/_templates/field-group-template/field-group-template.component.ts

```diff
@@ -30,9 +30,6 @@ export class FieldGroupTemplateComponent implements OnInit {
     menuIconOverride$: any;
     prevRefLength: number;
     ngOnInit(): void {
-        if (!this.PCore$) {
-            this.PCore$ = window.PCore;
-        }
```

Comment:
Any reason for deleted this code?

Reply:
yes, I have added the PCore in ngOnChanges method, Hence removed in ngOnInit method

Reply:
Was there a reason you had to move it there? PCore$ should always be set and shouldn't ever change when it gets to this point in the code and moving it to ngOnChanges makes the if get run a lot more often.

Reply:
yes John, ngOnChanges method using one of the PCore method and getting undefined error as ngOnchanges method will call before ngOnInit due to that move the PCore initialization to ngOnChanges.

And you are crct ngOnChanges will call more often every time it will reinitialize, I will add condition for that undefined error and update the PR